### ISSUE TITLE
Composer command in Dockerfile fails due to inconsistent version requirements

### ISSUE THREAD

**Issue by brianV** [php-pm/php-pm-docker#15]
The following composer commands fail when the image is built:

`composer require php-pm/php-pm:${version} && composer require php-pm/httpkernel-adapter:${http_version}`

By default, `php-pm/php-pm:${version}` requires the master branch. However, httpkernel-adapter requires `php-pm/php-pm: "^1.0"`. This creates a conflict as both versions can't be co-installed.

The 'simple' solution may be to just remove the composer call to install php-pm/php-pm and just let it pull it in as the php-pm/httpkernel-adapter dependency.

**Comment by TBK** [comment]
A workaround is to create a composer.json file with this:

```
{ 
    "repositories": [ 
        { 
            "type": "vcs", 
            "url": "https://github.com/php-pm/php-pm" 
        } 
    ], 
    "require": { 
        "php-pm/httpkernel-adapter": "dev-master", 
        "php-pm/php-pm": "dev-master as 1.0.2" 
    } 
} 
```

**Comment by marcj** [comment]
Indeed, that's why I build always with fixed released version numbers. Like so:

`VERSION=1.0.3 HTTP_VERSION=1.0.3 TAG=1.0.3 make nginx`

**Comment by andig** [comment]
Released 2.0.3

**Comment by billip215** [comment]
@andig the image didn't work for --debug=1, it should be fixed by this:
https://github.com/php-pm/php-pm-httpkernel/commit/e060e0b98c56f9cb0592b68961452a8392ff1f7e
Not sure if you should use "dev-master" on Dockerfile instead of released version.

**Comment by andig** [comment]
Thanks for the nudge. I have released httpkernel as 2.0.3 and will publish docker images as 2.0.3-2.
